### DISCUSSION TITLE
Basic setup question

### DISCUSSION THREAD

**Discussion by dcCMPY** [gtxaspec/wz_mini_hacks#344] [Q&A]
OK - having trouble here, when I shouldnt.

So firstly I'm unable to connect via SSH.

I generated public/private rsa key pair and entered a passphrase via command prompt using ssh-keygen

From the .pub file copied all contents except for the start and end of the file (ssh-rsa and the end xxx@computername)

to the wz_mini/etc/ssh/authorized_keys file

Can someone please let me know what I'm doing wrong?

Also I cannot seem to get RTSP streaming working

- I've copied all the contents under the SD_ROOT folder

- I've updated the .conf file to be below, login and password is different

```
RTSP_LOGIN="admin"

RTSP_PASSWORD="password"

RTSP_PORT="8554"

RTSP_HI_RES_ENABLED="true"

RTSP_HI_RES_ENABLE_AUDIO="true"

RTSP_HI_RES_MAX_BITRATE="2048"

RTSP_HI_RES_TARGET_BITRATE="1024"

RTSP_HI_RES_ENC_PARAMETER="2"

RTSP_HI_RES_FPS=""

RTSP_LOW_RES_ENABLED="false"

RTSP_LOW_RES_ENABLE_AUDIO="false"

RTSP_LOW_RES_MAX_BITRATE=""

RTSP_LOW_RES_TARGET_BITRATE=""

RTSP_LOW_RES_ENC_PARAMETER=""

RTSP_LOW_RES_FPS=""

RTSP_AUTH_DISABLE="false"
```

- When the camera boots, I can hear it mentioning the fact its loading what is on the SD card

- I can ping the camera

- Yet when I try to connect via Synology Surveillance Station using their manually method, I'm unable to connect

Is there anything else that I need to do or other sections in the .conf file that I need to update

**Comment by bjs-pdx** [comment]
Regarding RTSP did you review this:
```

Single Stream

the singular stream will be located at rtsp://login:password@IP_ADDRESS:8554/unicast
Multiple Streams

multiple streams are located at

    V3 - rtsp://login:password@IP_ADDRESS:8554/video1_unicast and rtsp://login:password@IP_ADDRESS:8554/video2_unicast
    V2 - rtsp://login:password@IP_ADDRESS:8554/video6_unicast and rtsp://login:password@IP_ADDRESS:8554/video7_unicast


```
With regard to ssh, did you get carriage returns and line feeds in the key file?  I did that at first.  What OS and app did you use to copy the data?

**Reply by dcCMPY** [reply]
Thank you for the reply :)

For RTSP, I think I got it. The below has worked, testing with VLC using this URL
 rtsp://xxxxx:xxxxx@192.168.1.179:554/unicast

```
RTSP_LOGIN="xxxxx"
RTSP_PASSWORD="xxxxx"
RTSP_PORT="554"

RTSP_HI_RES_ENABLED="true"
RTSP_HI_RES_ENABLE_AUDIO="true"
RTSP_HI_RES_MAX_BITRATE="2048"
RTSP_HI_RES_TARGET_BITRATE="1024"
RTSP_HI_RES_ENC_PARAMETER="2"
RTSP_HI_RES_FPS="20"

RTSP_LOW_RES_ENABLED="false"
RTSP_LOW_RES_ENABLE_AUDIO="false"
RTSP_LOW_RES_MAX_BITRATE=""
RTSP_LOW_RES_TARGET_BITRATE=""
RTSP_LOW_RES_ENC_PARAMETER=""
RTSP_LOW_RES_FPS=""

RTSP_AUTH_DISABLE="false"
```

However using my Synology NAS - Synology Surveillance Station, it still won't authenticate against the RTSP stream. Not sure what I'm doing wrong.

________________________________
For SSH access, I have also got this working. 

First off it was failing with _No supported authentication methods available (server sent: publickey)_ within Putty I added the private key that was generated earlier, under Connection>SSH>Auth and under Authentication parameters, browsed and selected the private key

I then opened PuttyGen and converted the private key correctly, then repeated the same steps adding the ppk properly.

Connection was successful after that :D
